diagnostic.md

```diff
@@ -5,74 +5,84 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The backend's role is to persist data used by you application.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The model is used to fetch data.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller communicates with the model
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+The view is the UI interface.  We use the MVC pattern using curl type request
+to simulate what the UI would send us.
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Creat Read Update Delete
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+Model
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+Index, Show Create Update, Destroy
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
+The routes.rb is look at to see if there is an action that supports the request.
+The route then calls the GET(show) metod in the people controller.
+The controller will then execute the methods that will send the 'find' command to the DB
+The model(DB) will respond to the controller
+The controller will look at the response and determine success/failure and send a response.
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+rails generate controller Greetings Hello
```

Comment:
Close, we would actually use `rails new <app_name>`

Your answer would only generate a controller, not a new app.